test-integration/scripts/24-tests-e2e.sh

```diff
@@ -20,7 +20,7 @@ fi
 #-------------------------------------------------------------------------------
 # Functions
 #-------------------------------------------------------------------------------
-launchCurlOrProtractor() {
+launchCurlOrCypress() {
```

Comment:
Maybe keep script compatible with protractor?
We might need to keep launch some tests using protractor... until we remove it.

Reply:
I have made modifications to be able to launch either Cypress or Protractor and add a Protractor build for Angular. Tell me wdyt